consensus/core/Cargo.toml

```diff
@@ -43,8 +43,10 @@ rustls.workspace = true
 serde.workspace = true
 shared-crypto.workspace = true
 strum_macros.workspace = true
+sui-config.workspace = true
 sui-macros.workspace = true
 sui-protocol-config.workspace = true
+sui-simulator.workspace = true
```

Comment:
This should be in `[target.'cfg(msim)'.dependencies]`, and potentially other additional dependencies.

Reply:
Or maybe it is better to have a `consensus-simulator` crate?